scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -65,16 +64,7 @@
     ]
 )
 
-# Support exception for latest central and sensor 3.74 as per
-# https://issues.redhat.com/browse/ROX-18223
-support_exceptions = [
-    ChartVersions(
-        central_version=latest_tag,
-        sensor_version=get_latest_helm_chart_version_for_specific_release(
-            "stackrox-secured-cluster-services", Release(major=3, minor=74)
-        ),
-    )
-]
```

Comment:
Could we at least have it commented out? Would be cool to have these tests once we fix the issue in 3.74

Reply:
Done